src/woo_publications/api/tests/test_utils.py

```diff
@@ -0,0 +1,26 @@
+from unittest import TestCase
+
+from ..utils import underscore_to_camel
+
+
+class UnderscoreToCamelTests(TestCase):
+    def test_single_word_does_not_get_manipulated(self):
+        result = underscore_to_camel("single")
+
+        self.assertEqual(result, "single")
+
+    def test_multiple_words_get_manipulated(self):
+        result = underscore_to_camel("multiple_words_get_manipulated")
+        self.assertEqual(
+            result,
+            "multipleWordsGetManipulated",
+        )
```

Comment:
I'm going to keep trying to hammer this in :smile: 

The visual separation with the blank line helps identifying the setup/act/assert blocks:

```suggestion
        result = underscore_to_camel("multiple_words_get_manipulated")

        self.assertEqual(
            result,
            "multipleWordsGetManipulated",
        )
```